managed-databases/postgresql-and-mysql/concepts.mdx

```diff
@@ -53,6 +53,16 @@ A Database Instance is a managed database service created upon a custom base ima
 
 A [snapshot](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) is a consistent, instantaneous copy of the Block Storage volume of your Database Instance at a certain point in time. They are designed to recover your data in case of failure or accidental alterations of the data by a user. They allow you to quickly create a new Instance from a previous state of your database, regardless of the size of the volume. Their limitation is that, unlike backups, snapshots can only be stored in the same location as the original data.
 
+## Encryption at rest
+
+Encryption at rest allows you to permanently encrypt your database data.
+
+When you enable encryption at rest, you cannot disable it after database creation. All databases, all data and all snapshots will be encrypted. Backup encryption is currently not available.
+
+During the beta phase, if chose to enable encryption after database creation, you must make an enablement request via a [support ticket](https://console.scaleway.com/support).
+
+Enabling encryption incurs in the copy of the database data into a new, encrypted block volume. This takes approximately 1h per 100GB of storage. A couple seconds of downtime are also expected towards the end of the copying process, when the volumes are being switched.
```

Comment:
There is something off about the first sentence here but I can't work out what it's meant to say?

Reply:
```suggestion
Enabling encryption means that your database data will be copied into a new, encrypted block volume. This takes approximately 1h per 100GB of storage. A couple seconds of downtime are also expected towards the end of the copying process, when the volumes are being switched.
```